control_plane/src/local_env.rs

```diff
@@ -72,6 +72,18 @@ pub struct PageServerConf {
 
     // jwt auth token used for communication with pageserver
     pub auth_token: String,
+
+    // GC activation period
+    pub gc_period: String,
+
+    // GC horizon
+    pub gc_horizon: u64,
+
+    // Checkpoint period
+    pub checkpoint_period: String,
+
+    // Checkpoint distance
+    pub checkpoint_distance: u64,
```

Comment:
Looks like this change is unrelated to failpoints. Could you extract this into a separate patch?

Reply:
This changes are needed for test_recovery.py test included in this PR.

Reply:
I'm trying to say that IMO failpoints infra patch should include neither the test nor those settings.

Reply:
IMHO it is strange to commit some feature without test.